appinventor/components-ios/src/extensions/posenet.swift

```diff
@@ -0,0 +1,255 @@
+// -*- mode: swift; swift-mode:basic-offset: 2; -*-
+// Copyright 2016-2023 MIT, All rights reserved
+// Released under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+import Foundation
+import WebKit
+
+
+open class PosenetExtension: NonvisibleComponent, WKScriptMessageHandler {
+  fileprivate final let _ERROR_WEBVIEWER_NOT_SET: String =
+  "You must specify a WebViewer using the WebViewer designer property before you can call"
+  fileprivate final let _ERROR_JSON_PARSE_FAILED: Int = 101
+  fileprivate final let _MODEL_URL: String =
+  "https://storage.googleapis.com/tfjs-models/savedmodel/posenet/mobilenet/quant2/050/"
+  fileprivate final let _BACK_CAMERA: String = "Back"
+  fileprivate final let _FRONT_CAMERA: String = "Front"
+  
+  fileprivate var _webview: WKWebView?  // Double check ! syntax
+  private var _webviewerApiSource: String = ""
+  private var _webviewerApi: WKUserScript? // double check ! syntax
+  
+  fileprivate var _keyPoints: [String: [String]] = [:]
```

Comment:
I feel like `_keyPoints` should be a `[String:[Double]]` since the values should be `x, y` pairs.